src/lib/ui/app/InsightCard/InsightCard.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import Profile from './Profile.svelte'
+  import ActionButton from './ActionButton.svelte'
+  import Tags from './Tags.svelte'
+  import Editorial from './Editorial.svelte'
+
+  const {
+    insight,
+  }: {
+    insight: {
+      title: string
+      user: { avatarUrl?: string; id: string; username: string }
+      tags: { name: string }[]
+      publishedAt: string
+      commentsCount: number
+      votes: {
+        currentUserVotes: number
+      }
+    }
+  } = $props()
+</script>
+
+<div class="rounded border border-porcelain p-6">
```

Comment:
It would be better to have a semantic markup for this component -> `article`, `section`, etc